ext/rbs_extension/location.c

```diff
@@ -54,27 +57,28 @@ static void check_children_cap(rbs_loc *loc) {
 }
 
 void rbs_loc_add_required_child(rbs_loc *loc, ID name, range r) {
-  check_children_cap(loc);
-
-  unsigned short i = loc->children->len++;
-  loc->children->entries[i].name = name;
-  loc->children->entries[i].rg = rbs_new_loc_range(r);
```

Comment:
This code is the same as `rbs_loc_add_optional_child`, so we can reuse that, and just set `required_p` after.